app/scripts/views/mixins/password-mixin.js

```diff
@@ -44,12 +37,8 @@ define(function (require, exports, module) {
           this.logViewEvent('password.visible');
         } else {
           passwordField.attr('type', 'password');
-          if (this.isPasswordAutoCompleteDisabled()) {
-            passwordField.attr('autocomplete', 'off');
-          } else {
-            passwordField.removeAttr('autocomplete')
-              .removeAttr('autocorrect').removeAttr('autocapitalize');
-          }
+          passwordField.removeAttr('autocomplete')      
+               .removeAttr('autocorrect').removeAttr('autocapitalize');
```

Comment:
@feelcoder up to you if you want to make this change or leave it like it is right now.